.github/workflows/assets.yml

```diff
@@ -47,7 +47,6 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
-          token: ${{ secrets.GITHUB_TOKEN_BOT }}
```

Comment:
If I remember correctly, the issue with the default is that when the PR is automatically created then CI could not run.

Reply:
Lets try it out. If it wont, there are few other options we can try.